http/handler_store_test.go

```diff
@@ -93,3 +94,79 @@ func TestExecRequest_WithInvalidQuery_HasSpecCompliantErrors(t *testing.T) {
 		"message": "Cannot query field \"invalid\" on type \"User\".",
 	}})
 }
+
+func TestExecRequest_WithValidQuery_HttpGet_WithOperationName_OmitsErrors(t *testing.T) {
+	cdb := setupDatabase(t)
+
+	query := `query {
+		User {
+			name
+		}
+	}`
```

Comment:
todo: Although the test does pass along the operation name in the query string, it doesn't test that it's actually passed to the GraphQL Request. To do so, the query might look like this
```go
query := `query UserQuery {
		User {
			name
		}
	}
        query UserQueryWithDocID {
		User {
                         _docID
			name
		}
	}`
```
And you can ensure that only the name field is returned to know that the operation name was correctly received. Even one operation might be enough to do so.

At the moment, this test will probably fail due to the operation not existing.

Reply:
Seems like you fixed that already so I'll resolve the comment.